### PR TITLE
ref: create cleanup

### LINKED ISSUE #10427: Deprecate and remove create
Time has come

# What
removing create as in: 
- https://github.com/kodadot/nft-gallery/issues/10390

-> not our product, our scope is gen art, not single nfts - we will introduce minter for generative drops

for now, there will be a placeholder page that will redirect traffic if anyone accidentally enters the page

# Task
- remove nft minter
- remove ai minter
- remove collection minter
- remove navbar link
- move navbar link to footer 
- add placeholder page
- save the code somewhere if we need it (really small probability) 

# Placeholder page
design updated here: https://www.figma.com/design/JXNtryaNYhY0XJNArzvPQx/Handoff?node-id=1287-18959&t=Dt9j4HyS2vIGJGpV-11

image

- super simple, just text and links
- form redirects here: https://form.kodadot.xyz/drop-interest
- documentation here: https://hello.kodadot.xyz/generative/generative-art

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10427

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI

Screenshot 2024-06-09 at 5 17 35 PM

### CODE REVIEW

**Comment by prachi00** [comment]
@exezbcz  about 'move navbar link to footer', where in footer? 


Also, are we saving the old code or not? if yes, where are we saving? keeping it as it is? or moving somewhere else?

**Comment by exezbcz** [comment]
image

divider missing, text to black please

**Comment by prachi00** [comment]
added create @exezbcz

**Comment by exezbcz** [comment]
> image
> 
> divider missing, text to black please

still this

**Comment by prachi00** [comment]
Screenshot 2024-06-11 at 11 38 10 AM
done

File: components/create/CreateLanding.vue [hunk 1]
```diff
@@ -1,202 +1,48 @@
...
-      @apply flex gap-8 text-left w-full;
-
-      &-text {
-        margin-top: 0;
-      }
-    }
-  }
-}
+<style scoped>
+/* Additional custom styles if needed */
```
Comment (Jarsen136) [inline components/create/CreateLanding.vue:47]: Could you remove it?

File: components/create/CreateLanding.vue [hunk 2]
```diff
@@ -1,202 +1,48 @@
...
+          >
+          To Learn More And Join Our Successful Drops.
+        </p>
+        <p class="">
+          For Code Structure Guidelines, Refer To Our
+          <a
+            href="https://hello.kodadot.xyz/generative/generative-art"
+            class="text-blue-500 underline"
+            >Documentation</a
+          >.
```
Comment (Jarsen136) [inline components/create/CreateLanding.vue:35]: Could you please add translation key?

File: components/common/ConnectWallet/ConnectWalletModal.vue [hunk 3]
```diff
@@ -3,6 +3,7 @@
     class="wallet-modal-container flex flex-col"
     data-testid="wallet-connect-sidebar-modal">
     <NeoModalHead
+      data-testid="kkkkkkkkkkkk"
```
Comment (Jarsen136) [inline components/common/ConnectWallet/ConnectWalletModal.vue:6]: 👀

Comment (prury) [reply components/common/ConnectWallet/ConnectWalletModal.vue:6]: ops, this was my doing

**Comment by exezbcz** [comment]
this is taking too long, what is left?

File: components/create/CreateLanding.vue (hunk 2 above)
Comment (prachi00) [reply components/create/CreateLanding.vue]: done

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
massmint is deleted as well?

**Comment by vikiival** [comment]
missing review from @hassnian

**Comment by prachi00** [comment]
> missing review from @hassnian

@Jarsen136 @vikiival massmint isn't visible but is accessible from url `/ahk/massmint/onboarding` , should we delete this whole flow then from code like now?

**Comment by hassnian** [comment]
> @prachi00 that is up to you, would be nice to have it somewhere in case we ever need it again, I guess github saves the versions

the whole NFT creation code is still there, if we are not planning on having it back any time soon , I'd just remove it all , we can always come back to this pr , we will always have version of the code with the create logic.  cc @Jarsen136 

> No Longer Support NFT Creation.

this means that you can't create any new nft via koda right ? cc @exezbcz 

create NFT still accessible `/ahk/create/nft` 

CleanShot 2024-06-15 at 09 50 30@2x

also massmint

**Comment by exezbcz** [comment]
> this means that you can't create any new NFT via koda right ? cc @exezbcz

yes

**Comment by exezbcz** [comment]
> the whole NFT creation code is still there, if we are not planning on having it back any time soon , I'd just remove it all , we can always come back to this pr , we will always have version of the code with the create logic. cc @Jarsen136
> 
> >

thought so, thank you

**Comment by prachi00** [comment]
@exezbcz are we deleting waifu stuff too?

**Comment by exezbcz** [comment]
> @exezbcz are we deleting waifu stuff too?

- https://github.com/kodadot/nft-gallery/issues/10427

only thing that is not mentioned there is removing the massmint. Thought removing things is faster then actually building it...

**Comment by exezbcz** [comment]
Hello, this PR has been open for 6 days and is still not finished. I appreciate your effort and look forward to your future contributions.

**Pull request closed without merging**